### PR TITLE
Feature/adjustable page gaps

### PR DESCRIPTION
**Pull request by h45h74x**
Hi there,

As I was a little annoyed by the gaps between pages, even when seleting the 'continuous' mode, I decided to make the gaps adjustable. You can now specify the gap size in px by adding the entry 'page_margin' to the config file. By setting the size to '0', you have no gaps at all.

Thank you for this great software!

_I created a new pull request and deleted the old one because I didn't read the development guideline first :man_facepalming:_

### CODE REVIEW

File: src/library/entry.cr [hunk 1]
```diff
@@ -117,6 +117,7 @@ class Entry
           sizes << {
             "width"  => size.width,
             "height" => size.height,
+            "margin" => Config.current.page_margin,
```
Comment (hkalexling) [inline src/library/entry.cr:120]: This method is supposed to return the width and height of all pages in an entry, and I don't think it makes sense to have the margin value here. Do you think you can remove this line and add it to the dimensions API instead? The API route is shown below:

https://github.com/hkalexling/Mango/blob/f7b8e2d852bd90b2130437c62f8f001451f557fc/src/routes/api.cr#L648-L669

Something like 

```cr
send_json env, {
  "success"    => true,
  "dimensions" => sizes,
  "margin" => Config.current.page_margin,
}.to_json
```

would do. You can then access the value from JS using `data.margin`.

**Comment by hkalexling** [comment]
Thanks a lot for the PR! It looks pretty good overall. I left a comment, and it would be great if you could revise a bit!

File: src/library/entry.cr (hunk 1 above)
Comment (h45h74x) [reply src/library/entry.cr]: Sure! I added the changes in this commit: https://github.com/hkalexling/Mango/pull/136/commits/aa707522449746aa88bc8f27b6e3107f2f976a2e

**Review by hkalexling — APPROVED** [review]

**Comment by hkalexling** [comment]
Great! Thanks for contributing 👍

**Pull request merged**